docs/configuration/plugins.md

```diff
@@ -33,3 +33,21 @@ Note that a plugin must be listed in `PLUGINS` for its configuration to take eff
 
 ---
 
+## PLUGINS_CONFIG
+
+Default: Empty
+
+This parameter holds configuration settings for how the individual plugins are displayed in the plugins table under Admin->System->Plugins.  Declaring `hidden` as True will make it so the plugin is not shown in the table.  Declaring `disabled` will make it show the plugin in the table, but it won't be linked to the details and installation page for the plugin. An example configuration is shown below:
+
+```python
+PLUGINS_TABLE_CONFIG = {
+    'plugin1': {
+        'hidden': True,
+    },
+    'plugin2': {
+        'disabled': True,
+    },
+}
```

Comment:
What do you think about flipping the ordering here?

```python
PLUGINS_TABLE_CONFIG = {
    'hidden': [
        'plugin1',
    ],
    'disabled': [
        'plugin2',
    ],
}
```

I ask because I can see us extending this configuration parameter in the future for other uses (e.g. disabling the catalog entirely or specifying an alternate URL), and preserving the root-level namespace would keep our options open.